lib/evss/intent_to_file/intent_to_file_response.rb

```diff
@@ -11,6 +11,10 @@ class IntentToFileResponse < EVSS::Response
       def initialize(status, response = nil)
         super(status, response.body) if response
       end
+
+      def cache?
+        ok? && !intent_to_file.expires_within_one_day?
```

Comment:
This is clever. We may have a similar need on Vet360 for doing less caching around updates. Nice to have an example to go off of.